velox/functions/sparksql/JsonObjectKeys.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/functions/prestosql/json/SIMDJsonUtil.h"
+
+namespace facebook::velox::functions::sparksql {
+
+/// json_object_keys(jsonString) -> array(string)
+///
+/// Returns all the keys of the outermost JSON object as an array if a valid
+/// JSON object is given.If it is any other valid JSON string, an invalid JSON
```

Comment:
nit: add space after `given.` Perhaps also update the comment to mention 3 cases instead of 4 to return NULL.

Reply:
Done